lang/ja.js

```diff
@@ -4,22 +4,22 @@ export default {
 	"components.alert.close": "アラートを閉じる",
 	"components.calendar.notSelected": "選択されていません。",
 	"components.calendar.selected": "選択されています。",
-	"components.calendar.show": "{month} の表示",
+	"components.calendar.show": "{month} を表示",
 	"components.dialog.close": "このダイアログを閉じる",
 	"components.input-date.clear": "クリア",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.openInstructions": "日付形式 {format} を使用します。ミニカレンダーにアクセスするには下矢印を使うか Enter を押します。",
+	"components.input-date.setToToday": "今日に設定",
 	"components.input-date-time.date": "日付",
-	"components.input-date-time.time": "時間",
+	"components.input-date-time.time": "時刻",
 	"components.input-search.clear": "検索のクリア",
 	"components.input-search.search": "検索",
-	"components.menu-item-return.return": "",
-	"components.menu-item-return.returnCurrentlyShowing": "{menuTitle}",
+	"components.menu-item-return.return": "前のメニューに戻ります。",
+	"components.menu-item-return.returnCurrentlyShowing": "前のメニューに戻ります。{menuTitle} を表示しています。",
 	"components.meter-mixin.commaSeperatedAria": "{term1}、{term2}",
 	"components.meter-mixin.fraction": "{x}∕{y}",
 	"components.meter-mixin.progressIndicator": "進捗状況インジケータ",
-	"components.more-less.less": "より少ない",
```

Comment:
"Fewer" -> "Cut back" and "Is more than" -> "Increase"... so maybe better?

Reply:
I think so? They make sense